.yamato/ruamel/build.py

```diff
@@ -64,6 +67,32 @@ def get_test_platforms(metafile_testplatforms):
 def get_agent(agent_name):
     return shared_agents[agent_name]
 
+def assert_dependencies():
+    for yml_file, yml_value in yml_files.items():
+        yml_content = yml_value['yml']
+        for job_id, job_content in yml_content.items():
+            for dependency in job_content.get('dependencies') or []:
+                dep_path = (dependency if  isinstance(dependency, str) else dependency['path']).split('/')[1]
+                dep_file, dep_job_id = dep_path.split('#')[0], dep_path.split('#')[1]
+                try:
+                    assert yml_files[dep_file]['yml'][dep_job_id] 
+                except:
+                    print(f'Mistake in file {yml_file}#{job_id} for dependency {dep_file}#{dep_job_id}')
+
+
+def add_comments():
+    for yml_file, yml_value in yml_files.items():
+        with open(os.path.join(save_dir,yml_value['path']), 'r+') as f:
+            yml = f.read()
+            f.seek(0, 0)
+            f.write(''' 
+# This file is generated by /ruamel/build.py. Don't edit this file manually. 
+# Introduce any changes under /ruamel folder within Python, and rerun build.py to regenerate all .yml files. 
+# Read more on the refactored Yamato docs. 
```

Comment:
I'm wondering if it would be useful to include the link to the docs here. It may be risky if the link changes later, but do the devs have another easy way to find it?

Reply:
i would be reluctant to include a google docs link in a public repo, but perhaps pin the doc to #devs-renderpipe and note that here?

Reply:
good one, i'll add the channel there